src/Inciter/Partitioner.h

```diff
@@ -387,6 +388,8 @@ class Partitioner : public CBase_Partitioner< HostProxy,
     //! \param[in] p The PE uniquely assigns the node IDs marked listed in ch
     //! \param[in] cn Vector containing the set of potentially multiple chare
     //!   IDs that we own (i.e., contribute to) for all of our node IDs.
+    //! \param[in] ce Map associating a vector of chare IDs to edges (at which
```

Comment:
What does this mean? Presumably this is what chares own vertices that we are connected to but that are not locally owned? What does initial mesh refinement have to do with this?